README.md

```diff
@@ -23,7 +23,7 @@ Create the virtualenv:
 
 Install the requirements:
 
-    pip install -r requirements.txt
+    sudo pip install -r requirements.txt
```

Comment:
I will not remove the virtualenv requirements.
 Virtualenv is a good idea for many reasons, first of all for security.
It is ok require the sudo for the system packages but it is not acceptable to install the python components.